server/channels/store/storetest/user_store.go

```diff
@@ -4488,6 +4489,39 @@ func testUserStoreAnalyticsGetInactiveUsersCount(t *testing.T, rctx request.CTX,
 	require.Equal(t, count, newCount-1, "Expected 1 more inactive users but found otherwise.")
 }
 
+func testUserStoreAnalyticsGetInactiveUsersCountIgnoreBots(t *testing.T, rctx request.CTX, ss store.Store) {
```

Comment:
0/5 This could also be a subtest of `testUserStoreAnalyticsGetInactiveUsersCount`.

Reply:
I preferred to keep them separate, but I'm also not strongly oppossed